.github/workflows/robot.yml

```diff
@@ -133,19 +133,17 @@ jobs:
           echo "bvt_coverage='bvt_coverage.out'" >> $GITHUB_ENV
           echo "ut_html_coverage='ut_coverage.html'" >> $GITHUB_ENV
           echo "bvt_html_coverage='bvt_coverage.html'" >> $GITHUB_ENV
-          echo "ut_excluded_pkg='pkg/pb\|pkg/sql/parsers/goyacc'" >> $GITHUB_ENV
-          echo "bvt_excluded_pkg='pkg/pb'" >> $GITHUB_ENV
+          echo "ut_excluded_pkg='pkg/pb\|pkg/sql/parsers/goyacc\|yaccpar'" >> $GITHUB_ENV
+          echo "bvt_excluded_pkg='pkg/pb\|yaccpar'" >> $GITHUB_ENV
```

Comment:
what's yaccpar

Reply:
I think this is auto generated by mysql parser, which will be deleted after build. But it will also be present in the outputted coverage file which will cause an error when generating the html, so need to exclude it